### PR TITLE
✨💚 [CI] provide node in build.sh and test build.sh on ubuntu

### PR DESCRIPTION
**Pull request by jxn-30**
**What kind of update does this PR provide?** *Please check*

- [ ] new translations / updated translations / translation fixes
- [x] a new feature
- [ ] a new module
- [ ] a bugfix for an existing feature / module
- [ ] improvement of style or user experience
- [ ] other: *Please fill out*

**What is included in your update?**
* includes installation of required `node` via [`nvm`](https://github.com/nvm-sh/nvm) in the build script `build/build.sh`

**Additional notes**
Please add any further notes here!

### CODE REVIEW

File: .github/workflows/test_build.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        name: '[🧪] Test Build on ${{ matrix.os }}'
+        steps:
+            - name: '[📥] checkout'
+              uses: actions/checkout@v3
+              with:
+                  ssh-key: ${{ secrets.SSH_DEPLOY_KEY }}
+                  ref: ${{ inputs.ref }}
+                  fetch-depth: 0
+
+            - name: '[👷] run build script'
```
Comment (Cr4zyc4k3) [inline .github/workflows/test_build.yml:28]: This would deploy in this scenario

File: scripts/createBuildScript.ts [hunk 2]
```diff
@@ -73,10 +101,19 @@ ${Object.entries(shortcuts)
     shift
 done`,
         'total_start_time=$(date +%s%N)',
+        "NODE_VERSION=$(grep '\"node\":' ./package.json | awk -F: '{ print $2 }' | sed 's/[\",]//g' | sed 's/\\^v//g' | tr -d '[:space:]')\n" +
```
Comment (Cr4zyc4k3) [inline scripts/createBuildScript.ts:104]: Bash escaped in .ts is pita to read

File: .github/workflows/test_build.yml (hunk 1 above)
Comment (jxn-30) [reply .github/workflows/test_build.yml:28]: the build script doesn't deploy or did I miss something?

File: scripts/createBuildScript.ts (hunk 2 above)
Comment (jxn-30) [reply scripts/createBuildScript.ts:104]: any ideas on how to improve that?

File: .github/workflows/test_build.yml (hunk 1 above)
Comment (Cr4zyc4k3) [reply .github/workflows/test_build.yml:28]: nvm, my bad

File: scripts/createBuildScript.ts (hunk 2 above)
Comment (Cr4zyc4k3) [reply scripts/createBuildScript.ts:104]: Maybe another shell-script which is called by the typescript-script? :D

Comment (jxn-30) [reply scripts/createBuildScript.ts:104]: seems like a little overload, doesn't it? :sweat_smile:

**Review by Cr4zyc4k3 — APPROVED** [review]

**Pull request merged**